SayTheirNames/Resources/Generated/strings.swift

```diff
@@ -91,12 +91,12 @@ internal enum L10n {
     internal static let age = L10n.tr("Localizable", "person.age")
     /// Children
     internal static let children = L10n.tr("Localizable", "person.children")
-    /// Social Media Hashtags
-    internal static let hashtags = L10n.tr("Localizable", "person.hashtags")
-    /// Press and hold to copy
-    internal static let hashtagCopyTip = L10n.tr("Localizable", "person.hashtagCopyTip")
     /// Copied
     internal static let copied = L10n.tr("Localizable", "person.copied")
+    /// Press and hold to copy
```

Comment:
Note : for some reason SwiftGen is just reordering those properties. Not sure why it like that better, but figured everyone had the same issue (this is a 0-sum game, i.e. no real changes)